mini-cross-build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
-for spark_version in 2.4.8 3.0.0 3.0.1 3.0.2 3.1.1 3.1.2 3.1.3 3.2.0 3.2.1 3.2.2 3.2.3 3.2.4 3.3.0 3.3.1 3.3.2 3.4.0 3.4.1 3.4.2 3.5.0 3.5.1 3.5.2 3.5.3 4.0.0-preview1 4.0.0-preview2
+for spark_version in 2.4.8 3.0.0 3.0.1 3.0.2 3.1.1 3.1.2 3.1.3 3.2.0 3.2.1 3.2.2 3.2.3 3.2.4 3.3.0 3.3.1 3.3.2 3.4.0 3.4.1 3.4.2 3.5.0 3.5.1 3.5.2 3.5.3 3.5.4 3.5.5 4.0.0-preview1 4.0.0-preview2
```

Comment:
Spark 3.5.6 is also released
```suggestion
for spark_version in 2.4.8 3.0.0 3.0.1 3.0.2 3.1.1 3.1.2 3.1.3 3.2.0 3.2.1 3.2.2 3.2.3 3.2.4 3.3.0 3.3.1 3.3.2 3.4.0 3.4.1 3.4.2 3.5.0 3.5.1 3.5.2 3.5.3 3.5.4 3.5.5 3.5.6 4.0.0-preview1 4.0.0-preview2
```